src/crimson/os/seastore/root_block.h

```diff
@@ -38,6 +38,25 @@ struct root_block_t {
   }
 };
 
+/**
+ * RootBlock
+ *
+ * Holds the physical addresses of all metdata roots.
```

Comment:
Probably not.  RootBlock doesn't deal with any logically addressed blocks at the moment.  You could, for instance, during mkfs create a block at *logical* address 0x0 to be the onode tree root (or a kind of logical root that contains the logical addr for the onode tree root).  This root is needed because it's not possible to have a stable physical address.